did/ion/testutil_test.go

```diff
@@ -13,8 +13,8 @@ var (
 	testData embed.FS
 )
 
-// RetrieveTestVectorAs retrieves a test vector from the testdata folder and unmarshals it into the given interface
-func RetrieveTestVectorAs(t *testing.T, fileName string, output interface{}) {
+// retrieveTestVectorAs retrieves a test vector from the testdata folder and unmarshals it into the given interface
+func retrieveTestVectorAs(t *testing.T, fileName string, output interface{}) {
```

Comment:
This function looks like it's repeated in another test. Could this be DRY'd up?

Reply:
not in an elegant way - no because `_test.go` files cannot be shared across packages. and we shouldn't bundle test code in the sdk